### PR TITLE
[NUI]public the converter for Xaml

### PR DESCRIPTION
**Pull request by AdunFang**
### Description of Change ###

Public the converter for Xaml.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by huayongxu — APPROVED** [review]
lgtm.

**Comment by huiyueun** [comment]
Why do you change to hidden api?

File: src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs [hunk 1]
```diff
@@ -554,7 +554,13 @@ bool CanBeSetFromStyle(BindableProperty property)
...
         }
 
-        internal void SetDynamicResource(BindableProperty property, string key)
+        /// <summary>
+        /// Internal use, will never open
+        /// </summary>
+        /// <param name="property"></param>
+        /// <param name="key"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetDynamicResource(BindableProperty property, string key)
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs:563]: Could you let us know why this method should change to public one?
Also, it'd better write a brief summary of this API.

**Review by dongsug-song — COMMENTED** [review]
Please fix the warnings shown below.
src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs#L100
src/Tizen.NUI/src/internal/XamlBinding/PositionTypeConverter.cs#L120
src/Tizen.NUI/src/internal/XamlBinding/RectangleTypeConverter.cs#L53
src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs#L51
src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs#L56
src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs#L67
src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs#L70

**Pull request merged**